st.clab.yml

```diff
@@ -99,15 +99,17 @@ topology:
 
     grafana:
       kind: linux
-      mgmt_ipv4: 172.80.80.43
-      image: grafana/grafana:8.5.2
+      mgmt-ipv4: 172.80.80.43
+      image: grafana/grafana:9.5.2
       binds:
         - configs/grafana/datasource.yml:/etc/grafana/provisioning/datasources/datasource.yaml:ro
         - configs/grafana/dashboards.yml:/etc/grafana/provisioning/dashboards/dashboards.yaml:ro
         - configs/grafana/dashboards:/var/lib/grafana/dashboards
-        - configs/grafana/grafana-flowcharting:/var/lib/grafana/plugins/grafana-flowcharting
       ports:
         - 3000:3000
+      exec:
+        - wget https://algenty.github.io/flowcharting-repository/archives/agenty-flowcharting-panel-1.0.0d.220606199-SNAPSHOT.zip -O /tmp/agenty-flowcharting-panel.zip
+        - unzip -d /var/lib/grafana/plugins/ /tmp/agenty-flowcharting-panel.zip 
```

Comment:
this is better to be changed to 

```
grafana-cli --pluginUrl https://algenty.github.io/flowcharting-repository/archives/agenty-flowcharting-panel-1.0.0d.220606199-SNAPSHOT.zip plugins install *
```

Also does grafana restart automatically after it senses the presence of a newly installed plugin? Or do you need to restart it manually?

Reply:
<img width="1914" alt="image" src="https://github.com/srl-labs/srl-telemetry-lab/assets/5679861/a6806324-25ad-4df7-9759-e3f47e7e5f59">
I just deployed it, and the panel is not available
I think you need to add a command to restart grafana after installation of a plugin

Reply:
That's weired.

Reply:
I did this now
sudo clab destroy --cleanup ; sudo clab deploy --reconfigure
then I used a different browser from the previous tests.
![image](https://github.com/srl-labs/srl-telemetry-lab/assets/1122166/cd6cb1a0-83ad-4cb4-a1c0-726aa7646b89)
Everything is looking fine.

Reply:
the different way of loading the plugins worked [a2f2801](https://github.com/srl-labs/srl-telemetry-lab/pull/23/commits/a2f2801375ae5f7555ca712a91002336f2c7278c)